Fix streaming wording and example (#19984)
dotnet/AspNetCore.Docs: aspnetcore/signalr/streaming.md

> [!NOTE]
> Write to the `ChannelWriter<T>` on a background thread and return the `ChannelReader` as soon as possible. Other hub invocations are blocked until a `ChannelReader` is returned.
>

> > Wrap logic in a `try ... catch`. Capture any exceptions inside the `catch` if you want to flow the error and complete the `Channel` in a `finally` to make sure the hub method invocation is completed properly.

Reviewer: Recommend singular "exception" and "error" all the way thru. It might be a bit challenging to grok with the condition in the middle. See if the following works ...

```suggestion
> Wrap logic in a [`try ... catch` statement](/dotnet/csharp/language-reference/keywords/try-catch). If you want to flow an error and complete the `Channel` in the hub method invocation properly, capture the exception inside the `catch` block and write it in a [`finally` block](/dotnet/csharp/language-reference/keywords/try-catch-finally).
```

Author: Tried my hand at the wording, the emphasis should be on the complete in the finally block.

Reviewer: LGTM, except possibly the singular/plural errors/exceptions.

> If you want to flow any errors, capture them inside the `catch` block and write them in the finally block.

... or ...

> If you want to flow an error, capture it inside the `catch` block and write it in the finally block.

Something like either of those to either make all of the nouns singular or plural.

... and btw for a **_crazy Rex super nit_** :smile: ... pronouns cause a number of problems in technical writing, so I use them carefully ... I could even say that I use them 'rarely.' In this case, the subject is really clear, so I think they work well [error(s) ... "them" or "it" later in the same sentence].
